interface-definitions/container.xml.in

```diff
@@ -334,6 +334,26 @@
                 </properties>
                 <defaultValue>rw</defaultValue>
               </leafNode>
+              <leafNode name="propagation">
+                <properties>
+                  <help>Volume bind propagation</help>
+                  <completionHelp>
+                    <list>rprivate rshared</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>rprivate</format>
+                    <description>Volume bind-propagation rprivate</description>
```

Comment:
We should explain what is `rprivate` to the user.


<html>
<body>
<!--StartFragment-->

Propagation setting | Description
-- | --
shared | Sub-mounts of the original mount are exposed to replica mounts, and sub-mounts of replica mounts are also propagated to the original mount.
slave | similar to a shared mount, but only in one direction. If the original mount exposes a sub-mount, the replica mount can see it. However, if the replica mount exposes a sub-mount, the original mount cannot see it.
private | The mount is private. Sub-mounts within it are not exposed to replica mounts, and sub-mounts of replica mounts are not exposed to the original mount.
rshared | The same as shared, but the propagation also extends to and from mount points nested within any of the original or replica mount points.
rslave | The same as slave, but the propagation also extends to and from mount points nested within any of the original or replica mount points.
rprivate | The default. The same as private, meaning that no mount points anywhere within the original or replica mount points propagate in either direction.

<!--EndFragment-->
</body>
</html>

Reply:
Done!